### DISCUSSION TITLE
Lost it... was the thesis in past, present or future tense ;)

### DISCUSSION THREAD

**Discussion by JRBliekendaal** [JRBliekendaal/master-thesis#56] [Q&A]
@edzob this is a typical rule I always forget

Is it like:
- The introduction is present tense.
- Normally the methodology is future tense (because the research methodology still has to be executed)
- Rest is past tense because you already did it.

**Comment by JRBliekendaal** [comment]
Already found the answer... good old scribbr

https://www.scribbr.com/language-rules/tenses/

**Comment by edzob** [comment]
In the a4 with requirements from AMS there is noted the preferred tense usage. (in addition to the breakdown you already provided)
